.github/workflows/deploy.yml

```diff
@@ -16,7 +16,19 @@ jobs:
       # VERY IMPORTANT: if you run this on any branch other than master, it will
```

Comment:
just a heads up, you may want to consider deployments to staging to trigger off git tagging, that way it avoids any accidents and resolves the need for the disclaimer, and has the added benefit of immutability